crates/sui-graphql-rpc/src/data/pg.rs

```diff
@@ -104,6 +106,9 @@ impl<'c> super::DbConnection for PgConnection<'c> {
         Q: QueryId + QueryFragment<Self::Backend>,
     {
         query_cost::log(self.conn, self.max_cost, query());
+        let binding = query();
+        let debugged = debug_query(&binding);
+        println!("Query: {}", debugged);
```

Comment:
The call to `query_cost::log` should be doing this, as long as you enable `DEBUG` logs.

Reply:
I apply a branch consisting of these debug queries and I always forget to drop it before a review :facepalm:

Reply:
yeah i dont need it anymore, just a creature of habit